OAuth/Response/LinkedinUserResponse.php

```diff
@@ -38,20 +38,20 @@ public function getLastName()
     public function getProfilePicture()
     {
         // https://docs.microsoft.com/en-us/linkedin/shared/references/v2/profile/profile-picture
+        /** @var array<string, array<string|int, array>> $profilePicture */
```

Comment:
checking https://docs.microsoft.com/en-us/linkedin/shared/references/v2/profile/profile-picture

this seems not correct, or?

I think `$profilePicture` is actually `array<string, string|array<string, array>>`. 

But since we are only interested in `$profilePicture['displayImage~']` maybe `array<string, array<string, array>>` is also fine

Reply:
With simpler version failed in last lines of this method.

Reply:
What about `array<string, string|array<string, array<int, array<string, mixed>>>>` ? :blush: I think this is accurate